app/js/settings.js

```diff
@@ -85,7 +85,7 @@ const Settings = {
    * @see Settings.set
```

Comment:
Can you add an a value in `defaults` at the top of this file for the new `wrangleOption` setting?

    wrangleOption: 'WITH_DUPES',

Reply:
fixed this